pkg/model/awsmodel/iam.go

```diff
@@ -124,26 +125,64 @@ func (b *IAMModelBuilder) Build(c *fi.ModelBuilderContext) error {
 		}
 	}
 
+	iamSpec := b.Cluster.Spec.IAM
+	if iamSpec != nil {
+		for _, sa := range iamSpec.ServiceAccountExternalPermissions {
```

Comment:
Do we have code to delete the relevant AWS objects when items are removed from this list?

Reply:
No, as usual, kOps is quite bad at this. I am wondering if we need a more generic solution that. Maybe we can tag all `RenderAWS`-created resources with `kops.k8s.io/task=taskname` and then at the end of cloudup, remove all resources that match tasks that wasn't created.

I think we need to look at that separately.